server/db/schema.ts

```diff
@@ -10,79 +10,44 @@ import {
   serial,
   foreignKey,
   boolean,
+  primaryKey,
   varchar,
 } from "drizzle-orm/pg-core";
 import { createId } from "@paralleldrive/cuid2";
 
 import { relations, sql } from "drizzle-orm";
+import { AdapterAccount } from "next-auth/adapters";
 
 export const role = pgEnum("Role", ["MODERATOR", "ADMIN", "USER"]);
 
-export const session = pgTable(
-  "Session",
-  {
-    id: text("id")
-      .$defaultFn(() => createId())
-      .notNull()
-      .primaryKey()
-      .unique(),
-    sessionToken: text("sessionToken").notNull(),
-    userId: text("userId")
-      .notNull()
-      .references(() => user.id, { onDelete: "cascade", onUpdate: "cascade" }),
-    expires: timestamp("expires", {
-      mode: "string",
-      withTimezone: true,
-      precision: 3,
-    }).notNull(),
-  },
-  (table) => {
-    return {
-      sessionTokenKey: uniqueIndex("Session_sessionToken_key").on(
-        table.sessionToken,
-      ),
-    };
-  },
-);
-
-export const sessionRelations = relations(session, ({ one, many }) => ({
-  user: one(user, { fields: [session.userId], references: [user.id] }),
-}));
-
-export const account = pgTable(
-  "Account",
-  {
-    id: text("id")
-      .$defaultFn(() => createId())
-      .notNull()
-      .primaryKey()
-      .unique(),
-    userId: text("userId")
-      .notNull()
-      .references(() => user.id, { onDelete: "cascade", onUpdate: "cascade" }),
-    type: text("type").notNull(),
-    provider: text("provider").notNull(),
-    providerAccountId: text("providerAccountId").notNull(),
-    refreshToken: text("refresh_token"),
-    accessToken: text("access_token"),
-    expiresAt: integer("expires_at"),
-    tokenType: text("token_type"),
-    scope: text("scope"),
-    idToken: text("id_token"),
-    sessionState: text("session_state"),
-  },
-  (table) => {
-    return {
-      providerProviderAccountIdKey: uniqueIndex(
-        "Account_provider_providerAccountId_key",
-      ).on(table.provider, table.providerAccountId),
-    };
-  },
-);
+export const session = pgTable("session", {
+  id: text("id")
+    .$defaultFn(() => createId())
+    .notNull()
+    .primaryKey()
+    .unique(),
+  sessionToken: text("sessionToken").notNull(),
+  userId: text("userId")
+    .notNull()
+    .references(() => user.id, { onDelete: "cascade" }),
+  expires: timestamp("expires", { mode: "date" }).notNull(),
+});
 
-export const accountRelations = relations(account, ({ one, many }) => ({
-  user: one(user, { fields: [account.userId], references: [user.id] }),
-}));
+export const account = pgTable("account", {
```

Comment:
Copy and pasted these schemas from https://authjs.dev/getting-started/adapters/drizzle